sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementClientBase.cs

```diff
@@ -15,6 +17,24 @@ public abstract class CosmosDBManagementClientBase : ManagementRecordedTestBase<
         public ResourceGroupsOperations ResourceGroupsOperations { get; set; }
         public CosmosDBManagementClient CosmosDBManagementClient { get; set; }
 
+        // Vars for Cassandra tests - using an existing DB account, since Account provisioning takes 10-15 minutes
+        public string location = "West US";
+        public string resourceGroupName;
+        public string databaseAccountName = "db" + new Random().Next(10000);
```

Comment:
Why does this name need to be randomly generated?

Reply:
Because it should be globally unique like the resource group name.

Reply:
Why does this have to be the case? The RG is already random, so it shouldn't conflict with anything. What problem is this solving?

Reply:
I changed it to be "db12345". Having it be randomly generated was causing problems with the recording.